src/networkUtils.ts

```diff
@@ -258,6 +263,9 @@ export async function forkNetwork(chainInfo: ChainCloneData, options: NetworkOpt
     chain.threshold = 3;
     chain.lastRelayedBlock = await chain.provider.getBlockNumber();
     chain.constAddressDeployer = new Contract(chainInfo.constAddressDeployer, ConstAddressDeployer.abi, chain.provider);
+    // Delete the line below and uncomment the line after when we deploy create3Deployer
+    await chain.deployCreate3Deployer();
+    //chain.create3Deployer = new Contract(chainInfo.create3Deployer, Create3Deployer.abi, chain.provider);
```

Comment:
I understand that we don't have `Create3Deployer` contract on forked network yet, so we have to deploy it here. 

For now, to make it available, can you also uncomment this line and replace `chainInfo.create3Deployer` with the deployed address?

Reply:
`chain.deployCreate3Deployer()` automatically makes it available so we do not need to manually assign it again. So uncommenting 268 would do nothing.